packages/strapi-admin/admin/src/components/Users/LoginModalSection/index.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ModalSection } from 'strapi-helper-plugin';
+import { Padded } from '@buffetjs/core';
+import { Row } from 'reactstrap';
+import { useIntl } from 'react-intl';
+import loginSettingsForm from 'ee_else_ce/components/Users/ModalCreateBody/utils/loginSettingsForm';
+
+import Input from '../../SizedInput';
+import Wrapper from '../ModalCreateBody/Wrapper';
+
+const LoginModalSection = ({ isDisabled, modifiedData, onChange, formErrors }) => {
```

Comment:
Instead of passing a very specific object as an argument, in that scenario, what do you think of passing the props one by one?

`<LoginModalSection hasSSORegistration={...} roles={...} />` ?

Reply:
Same for errors?

Reply:
I don't really understand your comment here, I'm sorry